docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+- Logitech C270 HD Webcam
+- Logitech StreamCam
+
+## Dependencies
+The software has dependencies, including Git, Docker, Docker Compose, and assorted command line tools. Follow the instructions below to install any dependency that is not already installed.  
+
+### Install Git
+Install Git from the official repository as documented on the [Git SCM](https://git-scm.com/download/linux) site.
+
+1. Update installation repositories:
+   ```bash
+   sudo apt update
+   ```
+
+2. Add the Git repository:
+   ```bash
+   sudo add-apt-repository ppa:git-core/ppa -y
+   ```
+
+3. Install Git:
+   ```bash
+   sudo apt install git
+   ```
+
+### Install Docker
+Install Docker from the official repository as documented on the [Docker](https://docs.docker.com/engine/install/ubuntu/) site.
+
+### Verify Docker
+To enable running Docker commands without the preface of sudo, add the user to the Docker group. Then run Docker with the `hello-world` test.
+
+1. Create Docker group:
+   ```bash
+   sudo groupadd docker
+   ```
+   >NOTE: If the group already exists, `groupadd` outputs a message: **groupadd: group `docker` already exists**. This is OK.
+
+2. Add User to group:
+   ```bash
+   sudo usermod -aG docker $USER
+   ```
+
+3. Refresh the group:
+   ```bash
+   newgrp docker 
+   ```
+
+4. To verify the Docker installation, run `hello-world`:
+
+   ```bash
+   docker run hello-world
+   ```
+   A **Hello from Docker!** greeting indicates successful installation.
+
+   ```bash
+   Unable to find image 'hello-world:latest' locally
+   latest: Pulling from library/hello-world
+   2db29710123e: Pull complete 
+   Digest: sha256:10d7d58d5ebd2a652f4d93fdd86da8f265f5318c6a73cc5b6a9798ff6d2b2e67
+   Status: Downloaded newer image for hello-world:latest
+
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+   ...
+   ```
+
+### Install Docker Compose
+Install Docker from the official repository as documented on the [Docker Compose](https://docs.docker.com/compose/install/#install-compose) site. See the Linux tab. 
+
+1. Download current stable Docker Compose:
+   ```bash
+   sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+   ```
+   >NOTE: When this guide was created, version 1.29.2 was current.
+
+2. Set permissions:
+   ```bash
+   sudo chmod +x /usr/local/bin/docker-compose
+   ```
+
+###  Download EdgeX Compose
+Clone the EdgeX compose repository
+
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+### Install Tools
+Install the build, media streaming, and parsing tools:
+
+   ```bash
+   sudo apt install build-essential vlc ffmpeg jq curl
+   ```
+
+NOTE: The device service ONLY works on Linux with kernel v5.10 or higher.  
+NOTE: An additional package may be required to build the device-use-camera service.  
+```
+sudo dpkg -i linux-libc-dev_5.10.0-14.15_amd64.deb
```

Comment:
Very specific version. Is this still needed? If not, consider removing.

Reply:
its in the readme, i can remove it from here.